README.md

```diff
@@ -12,14 +14,19 @@ The Open Neural Network Exchange implementation in MLIR (http://onnx.ai/onnx-mli
 | amd64-Windows | [![Build Status](https://dev.azure.com/onnx-pipelines/onnx/_apis/build/status/MLIR-Windows-CI?branchName=main)](https://dev.azure.com/onnx-pipelines/onnx/_build/latest?definitionId=9&branchName=main)             |
 | amd64-macOS   | [![Build Status](https://github.com/onnx/onnx-mlir/actions/workflows/macos-amd64-build.yml/badge.svg)](https://github.com/onnx/onnx-mlir/actions/workflows/macos-amd64-build.yml)             |
 
+This project contributes an ONNX Dialect that can be integrated in other projects, a compiler interfaces that lower ONNX graphs into MLIR files/LLVM bytecodes/C & Java libraries, an `onnx-mlir` driver to perform these lowering, and a python/C/C++/Java runtime environment.
```

Comment:
This project contributes:
- an ONNX Dialect that can be integrated in other projects
- a compiler interface that lowers ONNX graphs into MLIR files/LLVM bytecode/C & Java libraries
- an `onnx-mlir` driver to perform these lowering, and a python/C/C++/Java runtime environment.

Reply:
make sense